curated_transformers/tokenization/sentencepiece_adapters.py

```diff
@@ -10,6 +12,8 @@
 _SPP_EOS = 2
 _SPP_UNK = 0
 
+RemoveBosEosInOutT = Union[List[Ragged], List[List[Ragged]], TransformerModelOutput]
```

Comment:
Same question about this type, don't we always have `TransformerModelOutput`?

Reply:
Same answer as above.